src/main/kotlin/app/Config.kt

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app
+
+import app.utils.Options
+
+/**
+ * Config data class.
+ */
+data class Config (
+        var username: String = "",
+        var password: String = "",
+        var repos: MutableList<Repo> = mutableListOf<Repo>()
+) {
+    fun setValue(key: String, value: String) {
+        when(key) {
+            "username" -> username = value
+            "password" -> password = value
+        }
```

Comment:
If you need some field specific logic, make it as separate function for that field